### ISSUE TITLE
I want to use new features form Selenide 6.x with Selenium 3.14

### ISSUE THREAD

**Issue by vinogradoff** [selenide/selenide#1657]
Please support a way to get the new features and fixes from Selenide 6.x (Selenium 4-version) for Selenium 3.14 users (supported in Selenide 5.x)

**Comment by mcahornsirup** [comment]
Interesting topic. We have issues with Selenium4 running in parallel... and therefore we wanted to do the same thing => Selenide 6.x + Selenium 3.x. + JUnit5.x ... But, maybe it is a better approach to stick to Selenide 5.x at the moment and leave Selenide 6.x exclusivly for Selenium 4.x

**Comment by asolntsev** [comment]
@mcahornsirup We were thinking about supporting Selenide6 + Selenium 3 as a paid feature. Is your company ready to pay for such features? ;)

Generally, I recommend to find out what is the problem with parallel running. I know there is a solution.

**Comment by mcahornsirup** [comment]
@asolntsev In general, I think we all want to use selenium4 anyway. For us, using selenium6 + selenium3 is a temporary solution until the parallelisation thing is fixed. But this is a selenium4 issue. 

From what we have seen, it is the usage of computable futures within selenium4 in conjunction with JUnit5. The defined number of threads being used is ignored and tests are going to be started right away without waiting for the others to be finished.

Payments/Support - One time fees are the most simple to organise ;) I am talking to my manager ...

**Comment by asolntsev** [comment]
Yes, I was reading about that threading issue. As much as I understand, Selenium maintainers claim it's not their issue and not going to fix it. Am I right?

**Comment by mcahornsirup** [comment]
I guess that more and more people are going to run into that issue and at some point a solution needs to address this. We could argue that it is a JUnit5 issue and not a Selenium issue ... but at the end the combination of Selenium4 + JUnit5 will spread - and the issue :)

I just talked to a dev collegue and we think that a clean separation should be the way to go. Selenide 5.x should stick to Selenium 3.x and Selenide 6.x should stick to Selenium 4.x. 

Backporting features from Selenide 6.x to Selenide 5.x seems to be a nice to have, but it might be a waste of tme too.

**Comment by asolntsev** [comment]
it seems the problem with parallel threads will be fixed in JUnit 5.9: https://github.com/SeleniumHQ/selenium/issues/10113#issuecomment-1003686452
